site_scons/prereq_tools/base.py

```diff
@@ -774,7 +776,14 @@ def _setup_build_type(self):
 
     def _setup_intelc(self):
         """Setup environment to use intel compilers"""
-        env = self.__env.Clone(tools=['intelc'])
+        try:
+            env = self.__env.Clone(tools=['doneapi'])
```

Comment:
what is doneapi?  A "daos" version of the oneapi scons interface?

Reply:
yeah, I added the d in case Intel creates and adds a real oneapi tool to scons so as to avoid any conflicts